tools/setup/setup_venvs.py

```diff
@@ -15,11 +15,14 @@
 
 PY2_DEV_REQS_FILE = os.path.join(ZULIP_PATH, "requirements", "py2_dev.txt")
 PY3_DEV_REQS_FILE = os.path.join(ZULIP_PATH, "requirements", "py3_dev.txt")
+THUMBOR_REQS_FILE = os.path.join(ZULIP_PATH, "requirements", "thumbor.txt")
 
 PY2 = sys.version_info[0] == 2
 
 def main(is_travis=False):
     # type: (bool) -> None
+    setup_virtualenv("/srv/thumbor-venv", THUMBOR_REQS_FILE, patch_activate_script=True)
```

Comment:
For the sake of parity in discussion here, I would like to post an update on this here as well.
So basically problem with addressing this is that thumbor doesn't support python 3 yet as per https://github.com/thumbor/thumbor/issues/738

So for now it was [decided](https://chat.zulip.org/#narrow/stream/backend/subject/thumbor/near/264744) that we just keep thumbor in different venv and make sure its interfacing with zulip is py3 compatible and let thumbor live in py2.